templates/list_pages/node--oe-list-page--full.html.twig

```diff
@@ -24,15 +25,11 @@
     <div class="row">
       <div class="col-md-6 col-lg-8 align-self-center">
```

Comment:
I think we only need these wrappers and classes if we also have a sort element. Which currently we don't.